app/views/questions/_fields.html.erb

```diff
@@ -28,6 +28,10 @@
     <% end %>
   </div>
 
+  <div>
+    <%= f.field(:audio_prompt, type: :file, hint: t("activerecord.hints.question.audio_prompt")) %>
```

Comment:
You shouldn't need to specify the hint location explicitly. See `option_set_id` above for instance. Remember, [convention over configuration](https://en.wikipedia.org/wiki/Convention_over_configuration)